Packs/SMIME_Messaging/Integrations/SMIME_Messaging/SMIME_Messaging.yml

```diff
@@ -70,6 +70,12 @@ script:
       name: encrypt_message
       required: true
       secret: false
+    - default: false
+      description: The encoding code to use when decode the message body, e.g 'ISO-8859-2''
```

Comment:
let's find a link to add to the desc with all available encodings, e.g: https://docs.python.org/3/library/codecs.html#standard-encodings